### DISCUSSION TITLE
Make SkeletonModification and SkeletonModificationStack local to scene by default

### DISCUSSION THREAD

**Discussion by HatAndTie** [godotengine/godot-proposals#10944] [2D]
The Skeleton Modifications work as resources, which is good for reusing them. Generally Resources are fine to be shared by all instances as they only hold information.
But since the modifications hold direct references to the bones they affect, they stop working as intended when instancing the scene multiple times, unless setting it and the stack both als local to scene.

This can be cumbersome having to do with every modifier, so either changing the way they reference the bones or making local to scene true by default would be a big QoL improvement.

**Comment by Calinou** [comment]
Do they always *need* to be unique in all use cases (e.g. if you have hundreds of characters all playing the same animation at the same time)? If not, I wouldn't change the default value for performance reasons (unique resources need more memory).

**Reply by HatAndTie** [reply]
I looked into all the unique Modifications to check. But then I realised that they don't actually store the bones directly and instead use node paths, just like animations. so it being local to scene shouldn't matter.
So the behaviour seems to be an bug actually.
